src/servers/main.cc

```diff
@@ -324,6 +325,10 @@ std::vector<Option> options_
       {OPTION_BACKEND_DIR, "backend-directory", Option::ArgStr,
        "The global directory searched for backend shared libraries. Default is "
        "'/opt/tritonserver/backends'."},
+      {OPTION_BUFFER_MANAGER_THREAD_COUNT, "buffer-manager-thread-count",
+       Option::ArgInt,
+       "The number of threads used to accelerate copies and other operations "
+       "required to manage input and output tensor contents. Default is 1."},
```

Comment:
I was incorrect before. The default here should be zero so that we don't create any worker threads unless the user explicitly requests it.

Reply:
That's right. Although CopyBuffer would not be triggered we should not have these threads created by default. Fixed!